__tests__/app.test.js

```diff
@@ -61,7 +60,7 @@ describe("/api/articles/:article_id", () => {
       .get("/api/articles/2")
```

Comment:
these tests can be chained together, as you can follow your `expect(200)` with a `.then((response) => {`